api/NICE.NextWeb.API/Startup.cs

```diff
@@ -38,12 +44,33 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 app.UseDeveloperExceptionPage();
             }
+            else
+            {
+                app.UseExceptionHandler(
+                    options =>
+                    {
+                        options.Run(async context =>
+                        {
+                            context.Response.StatusCode = (int)HttpStatusCode.BadRequest;
```

Comment:
Generally this would be a 500 not a 400, so I think HttpStatusCode.InternalServerError makes more sense
```suggestion
                            context.Response.StatusCode = (int)HttpStatusCode.InternalServerError;
```